vars/withParameterizedPipeline.groovy

```diff
@@ -32,7 +32,7 @@ def call(type, String product, String component, String environment, String subs
   assert pipelineType != null
 
   Builder builder = pipelineType.builder
-
+  Environment environment = new Environment(env)
```

Comment:
Prob don't want to add another `environment` variable to the scope - this is just going to confuse people in the future, particularly renaming the existing variable to make this fit in.

```suggestion
  def pactBrokerUrl = (new Environment(env)).pactBrokerUrl
```